templates/copyright.tex

```diff
@@ -1,4 +1,4 @@
 
 \noindent \rule{\textwidth}{1pt}
 
-©2023 PingCAP All Rights Reversed.
\ No newline at end of file
+©2023 PingCAP All Rights Reserved.
```

Comment:
Let's keep it consistent with that on the PingCAP official website https://www.pingcap.com/tidb/
```suggestion
© 2023 PingCAP. All Rights Reserved.
```